docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,96 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+The :ref:`Flocker Installer <labs-installer>` allows you install Flocker in a variety of environments.
+For some of these environments, such as CoreOS, you can actually install Flocker inside a container.
+
+.. note::
+   Installing Flocker in containers is useful for environments such as CoreOS, RancherOS, or Amazon ECS, but please note that doing so is currently experimental.
+
+You can find the relevant Docker Hub images here:
+
+* `Dataset Agent <https://hub.docker.com/r/clusterhq/flocker-dataset-agent/>`_
+* `Container Agent <https://hub.docker.com/r/clusterhq/flocker-container-agent/>`_
+* `Control Service <https://hub.docker.com/r/clusterhq/flocker-control-service/>`_
+* `Docker Plugin <https://hub.docker.com/r/clusterhq/flocker-docker-plugin/>`_
+
+Before you install Flocker in containers, all authentication must be completed.
+This means that you will have generated the cluster certificate and key, the  node certificates and keys, and the API certificates and keys.
```

Comment:
I think this bit needs more description.
When I've been working with Giant Swarm, they've been using separate containers to hold the certificates and configuration for each Flocker service and fixing the UUID of the generated node certificate.
And there are some tools in the flocker-coreos repo to support that by monkey patching `flocker-ca`:
 * https://github.com/ClusterHQ/flocker-coreos/blob/master/flocker-bits/flocker-core/flocker-ca
See also: 
 * https://github.com/ClusterHQ/flocker-coreos/tree/master/flocker-bits#certs